scripts/rhel_validation/prepare_lis_vm.sh

```diff
@@ -21,6 +21,7 @@ function prepare_vm {
     kernel_version="$3"
     rhel_username="$4"
     rhel_password="$5"
+    storage_sas_token="$6"
```

Comment:
a new mandatory?

Reply:
Yes it is mandatory to upload the kernel to blob